packages/button/components/ButtonBase.tsx

```diff
@@ -66,6 +62,7 @@ export interface ButtonProps {
    * the type of button - same as HTML "type" attribute on the <button> tag
    */
   type?: "button" | "reset" | "submit";
+  ref?: any;
```

Comment:
is this really any?

Reply:
Wouldn't `ref` be stored on the class and not the props anyway? you need to create the ref when rendering. I think the type would be `React.RefObject<HTMLButtonElement>`?

[Good article on React refs & TS](https://medium.com/@martin_hotell/react-refs-with-typescript-a32d56c4d315)

Reply:
I tried that type and it didn't work. The ref is actually created in `FocusStyleManager`. I'll see what I can do to get rid of the `any`. Thanks for the article!